src/Movie-details/MovieDetails.js

```diff
@@ -89,21 +111,21 @@ class MovieDetails extends Component {
     if (date[0].charAt(0) === '0') date[0] = date[0].slice(1)
     if (date[1].charAt(0) === '0') date[1] = date[1].slice(1)
     return date.join('/')
-	}
-	
+  }
+
   render() {
     return (
       <>
-				{this.state.error && <h2>{this.state.error}</h2>}
+        {this.state.error && <h2 className='error'>{this.state.error}</h2>}
```

Comment:
As discussed, if we have time we can go back and add some styling to this error message so it's easier on the eyes of the user!